pep/app/pods/admin/general/controller.ts

```diff
@@ -75,12 +134,38 @@ export default class AdminGeneral extends Controller {
         this.changeset?.set('configSettings.home.expertPicks', [...filteredPicks]);
     }
 
+    /**
+     * Delete the video
+     *
+     * @param {VideoConfiguration} video
+     * @memberof AdminGeneral
+     */
+    @action
+    deleteVideo(video: VideoConfiguration): void {
+        const videos: VideoConfiguration[] = this.changeset?.get('configSettings.global.cards.videoPreviews');
+        const newVideos = videos.filter((item) => item !== video);
+        this.changeset?.set('configSettings.global.cards.videoPreviews', [...newVideos]);
```

Comment:
```suggestion
        this.changeset?.set('configSettings.global.cards.videoPreviews', newVideos);
```
no need to spread to a new array here, `videos.filter()` already returns a new array.